src/ganj/GuessANumberJava.java

```diff
@@ -0,0 +1,66 @@
+package ganj;
+
+import java.util.Random;
+import java.util.Scanner;
+
+public class GuessANumberJava {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        mainMenu();
+        String input = scanner.nextLine();
+        switch (input) {
+            case "1":
+                againstAI(scanner);
+                break;
+            case "2":
+                System.out.println("Soon...");
+                break;
+            case "3":
+                help();
+                break;
+            default:
+                System.out.println("Wrong input");
+                break;
+        }
+
+
+    }
+
+    private static void mainMenu() {
+        System.out.println("Welcome to Guess A Number game");
+        System.out.println("Choose o game mode:");
+        System.out.println("1 - AI v Player");
+        System.out.println("2 - Player v AI");
+        System.out.println("3 - How to play");
+        System.out.println("4 - Close app");
+    }
+
+    private static void againstAI(Scanner scanner) {
+        Random random = new Random();
+        System.out.println("Enter the range of numbers you want the target number to be between:");
```

Comment:
You ask for a range which is from 5 to 22 for example, but read only one number from keyboard. The message doesn't correspond to the actions.